src/main/scala/beam/agentsim/agents/vehicles/BeamVehicleType.scala

```diff
@@ -31,6 +31,7 @@ case class BeamVehicleType(
   chargingCapability: Option[ChargingPointType] = None,
   payloadCapacityInKg: Option[Double] = None
 ) {
+  def isSharedVehicle: Boolean = id.toString.startsWith("sharedVehicle")
```

Comment:
Method `beam.agentsim.agents.vehicles.BeamVehicle#isSharedVehicle` should call this method to have this code in one place.

Reply:
adjusted, let me know if I understood it wrongly